src/firmware_new/boards/arduino_uno/echo_library.cc

```diff
@@ -0,0 +1,20 @@
+
+#ifdef PLATFORMIO_BUILD
+#include "echo_library.h"
+#else
+#include "echo_library.h"
+#endif
```

Comment:
seems like we're doing the same thing regardless of the condition

Reply:
the first include will always be to the `file_name.h` , the second is to the relative path to the file (which in this case happens to be the same). Only the first one is necessary, but I think keeping the second would be a useful for the Arduino code to ease code navigation.

Reply:
is this so we can use this file outside of platformio?

Reply:
if we want to yes

Reply:
Well we probably don't need to support that. See the jupyter notebook example. But anyway, I don't really mind it as long as it's commented